docs/02-skog-context.md

```diff
@@ -59,6 +59,19 @@ function skogMiddleware(req, res, next) {
 }
 ```
 
+Lastly, sometimes there is already a context opened when calling this middleware (e.g., there is another middleware that opnens such context). To support it, we call `getFields` and add the returned object to the fields:
+
+```ts
+function skogMiddleware(req, res, next) {
+  const fields = {
+    ...getFields(),
+    req_id: nanoid(),
```

Comment:
This opens up to name conflicts (which is the major complaint with the context approach in the first place). Perhaps consider mentioning this or allow a way to handle naming conflicts.

Reply:
True. And returning to the analogy with React Context, it has the same problem. I will try to mention it in some understandable way...